app/models/draft.rb

```diff
@@ -7,6 +7,14 @@ class Draft < ActiveRecord::Base
 
   belongs_to :user
 
+  after_create do
+    UserStat.update_draft_count(self.user_id)
```

Comment:
There is a fair bit of duplication here so I think moving `UserStat.update_draft_count` will help DRY the code up.

Reply:
Also we can probably do something like

`after_commit :update_draft_count, on: [:create, :destroy]`